test/test_components.jl

```diff
@@ -58,13 +58,7 @@ add_comp!(my_model, testcomp1)
 
 # Testing to catch if before or after does not exist
 @test_throws ErrorException add_comp!(my_model, testcomp2, before=:testcomp3)
-
-# ArgumentError in v1.0-v1.2, ErrorException in v1.3
-if(VERSION < v"1.3.0")
-    @test_throws ArgumentError add_comp!(my_model, testcomp2, after=:testcomp3)
-else
-    @test_throws ErrorException add_comp!(my_model, testcomp2, after=:testcomp3)
-end
+@test_throws ErrorException add_comp!(my_model, testcomp2, after=:testcomp3)
```

Comment:
this will now fail v1.2 and v1.3 tests, could move to a separate PR if that's an issue